docs/nfl.html

```diff
@@ -0,0 +1,184 @@
+                padding: 1em !important;
+            }
+
+            ol.ui.list {
+                margin: 0 0 !important;
+            }
+        }
+
+        h1, span.subheading {
+            text-align: center;
+        }
+
+        #root {
+            background: #003399;
+        }
+
+        body, h1, .ui.button {
+            font-family: "Helvetica Neue", Helvetica, Arial, sans-serif !important;
+        }
+
+        #main {
+            display: flex;
+            flex-direction: column;
+            justify-content: space-between;
+            align-items: center;
+            text-align: left;
+            background: #FFFFFF;
+            max-width: 800px;
+            margin: 2em auto;
+            padding: 1.5em 3em;
+            box-shadow: 15px 15px 20px 0 hsla(0, 0%, 0%, 0.3);
+        }
+
+        #simulator {
+            position: relative;
+            width: 300px;
+            height: 380px;
+            overflow: hidden;
+            margin: 1em auto;
+        }
+
+        #simulator iframe {
+            position: absolute;
+            top: 0;
+            left: 0;
+            height: 380px;
+        }
+
+        .ui.button {
+            width: 14em;
+            padding: 1.5em;
+            margin: 2em auto;
+            background-color: #001A80;
+            color: #FFFFFF;
+            display: block;
+        }
+
+        .ui.button:focus, .ui.button:hover {
+            background-color: #003399;
+            color: #FFFFFF;
+        }
+
+        ol li::before {
+            color: #FFFFFF !important;
+            background-color: #57BB31 !important;
+            font-size: 2em;
+            font-weight: 600;
+
+            width: 2.5rem;
+            height: 2.5rem;
+            margin-left: -3rem !important;
+            margin-top: -.5rem !important;
+
+            padding-right: 0.7rem;
+            padding-top: .6rem;
+            border-radius: 50%;
+        }
+
+        ol.ui.list {
+            padding: 0 2em;
+            margin: 1em 1.25em;
+            align-self: stretch;
+        }
+
+        span.subheading {
+            line-height: 1.28571429;
+            margin-top: -.5em;
+            font-size: 1.75em;
+            font-weight: 700;
+            padding-bottom: 1em;
+        }
+
+        .logotext {
+            max-width: 30%;
+        }
+
+        /** footer styling **/
+        footer.ui * {
+            color: white !important;
+            opacity: 0.9;
+        }
+        .ui.divider:not(.vertical):not(.horizontal) {
+            border-top: 1px solid hsla(0, 0%, 100%, 0.3);
+            border-bottom: 1px solid hsla(0, 0%, 55%, 0.1);
+        }
+    </style>
+
+    <script>
+        function registerTickEvents() {
+            function tickEvent(id, data, measures) {
+                window.appInsights.trackEvent(id, data, measures);
+            }
+
+            document.querySelector("#openmod").addEventListener("click", function() {
+                tickEvent("nfl.open.mod");
```

Comment:
small: would rename these to "nfl.mod" / "nfl.tutorial" (or "nfl.mod.open" if you anticipate there being more than one type of "mod" event)